cmake/CmakeFunctionHelper.cmake

```diff
@@ -84,358 +84,100 @@ function(set_target_compile_def in_target)
     message(STATUS "Gemm vectorization support enabled for target ${in_target}")
     target_compile_definitions(${in_target} PUBLIC GEMM_VECTORIZATION_SUPPORT=1)
   endif()
-endfunction()
 
+  if(BLAS_ENABLE_CONST_INPUT)
+    target_compile_definitions(${in_target} PUBLIC BLAS_ENABLE_CONST_INPUT=1)
+  endif()
+endfunction()
 
 # blas unary function for generating source code
 function(generate_blas_unary_objects blas_level func)
```

Comment:
Maybe we can rename this function here to just `generate_blas_objects` instead of `generate_blas_unary_objects`. The name would be confusing for `binary` or `ternary` operators.

Reply:
Thank for addressing this